terminal.go

```diff
@@ -20,6 +22,12 @@ func colorEnabledWriter(w io.Writer) (io.Writer, bool) {
 	if !ok {
 		return w, false
 	}
+	// NO_COLOR is a relatively new standard for preventing color enabled
+	// writers rather than using the TERM env.
+	// See: https://no-color.org/
+	if truthy(os.Getenv("NO_COLOR")) {
```

Comment:
So the spec says:
```
Command-line software which adds ANSI color to its output by default should check for the presence of a NO_COLOR environment variable that, when present (regardless of its value), prevents the addition of ANSI color.
```

Should we be checking for Truthy at all?

Reply:
Yeah, I was thinking of changing this, so we can just say if it exists, then that's valid.